### PR TITLE
Material Dark Red theme

### PR DESCRIPTION
**Pull request by 95gabor**
image

### CODE REVIEW

File: material_dark_red.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  paper-slider-knob-start-color: "var(--primary-color)"
+  paper-slider-pin-color: "var(--primary-color)"
+  paper-slider-secondary-color: "var(--light-primary-color)"
+  paper-toggle-button-checked-ink-color: "var(--dark-primary-color)"
+  paper-toggle-button-checked-button-color: "var(--primary-color)"
+  paper-toggle-button-checked-bar-color: "var(--light-primary-color)"
+  paper-toggle-button-unchecked-bar-color: "var(--primary-text-color)"
+  secondary-text-color: "var(--primary-color)"
+  table-row-background-color: "var(--paper-card-background-color)"
+  table-row-alternative-background-color: "var(--sidebar-text_-_background)"
```
Comment (maartenpaauw) [inline material_dark_red.yaml]: @95gabor Could you please add a new line at the end of this file?

**Comment by maartenpaauw** [comment]
Hello @95gabor!

Thanks for adding you theme. It looks nice!
I've added some comments to both of the files.
Could you please fix that?

Once it is fixed I'll merge the pull request.

Thanks!

File: README.md [hunk 2]
```diff
@@ -12,6 +12,7 @@
 | Another Dark    | [lambtho](https://community.home-assistant.io/u/lambtho) posted [Another Dark Theme](https://community.home-assistant.io/t/another-dark-theme/28595)                 | dark                |
 | Halloween       | [skalavala](https://community.home-assistant.io/u/skalavala) posted [Halloween Theme](https://community.home-assistant.io/t/halloween-theme/30872)                   | halloween           |
 | Material Dark   | [matust](https://community.home-assistant.io/u/matust) posted [Material dark theme](https://community.home-assistant.io/t/material-dark-theme/30796)                 | material_dark_green |
+| Material Dark Red   | [95gabor](https://community.home-assistant.io/u/95gabor) posted [Material dark red theme](#)                 | material_dark_red |
```
Comment (maartenpaauw) [inline README.md]: @95gabor Did you submit your theme to the community forum? If so, you can put the link at the #.

**Comment by 95gabor** [comment]
> Hello @95gabor!
> 
> Thanks for adding you theme. It looks nice!
> I've added a comment to one of the files.
> Could you please fix that?
> 
> Once it is fixed I'll merge the pull request.
> 
> Thanks!

Hi @maartenpaauw!

I'm on it to fix it. Which comment do you mean?

Thanks!

**Comment by maartenpaauw** [comment]
Hello @95gabor,

The comments are listed above 😃.

**Review by maartenpaauw — CHANGES_REQUESTED** [review]

**Comment by maartenpaauw** [comment]
> Hi @maartenpaauw!
> 
> I'm on it to fix it. Which comment do you mean?
> 
> Thanks!

@95gabor Oops.. I forgot to start the review process. You should be able to see the comments right now.

File: material_dark_red.yaml (hunk 1 above)
Comment (95gabor) [reply material_dark_red.yaml]: @maartenpaauw new line added in this commit:  https://github.com/maartenpaauw/home-assistant-community-themes/pull/1/commits/39a3d257d49d93c14ae893b910be3e759b7d6d0d

File: README.md (hunk 2 above)
Comment (95gabor) [reply README.md]: @maartenpaauw I made a new topic for this theme. README link updated in https://github.com/maartenpaauw/home-assistant-community-themes/pull/1/commits/39a3d257d49d93c14ae893b910be3e759b7d6d0d

**Review by maartenpaauw — APPROVED** [review]

**Pull request merged**